CaseStudy.md

```diff
@@ -0,0 +1,166 @@
+  - The output of the Model Factory is one or more ML models packaged inside their own containerized environment (such as a Docker image).
+- #### <ins> Use Case Builder:</ins>
+   - The Use Case Builder provides a solid foundation for the Deployment processes and Pace by providing a convention based framework which enables the developers to build the orchestration layer responsible for interfacing with the underlying edge devices, peripherals and interacting with the ML Model containers for inferencing.
+   - It also enables other key features like end-to-end automated deployment validation of the use case on the targeted edge devices, packaging and publishing the validated orchestration code and edge deployment manifest to a central repository.
+- #### <ins>Use Case Deployment:</ins>
+   - Use case deployment enables rolling full and incremental deployments of the published use case to the edge devices leveraging the native IoT Edge capabilities.
+   - **Due to the customer's well-established maturity in edge deployment processes, this component will be developed by the customer and would seamlessly integrate with the Use Case builder to provide an end-to-end deployment experience.
+
+## Technical Implementation
+The following section provided an high-level overview of the key solution components.
+### Model Factory Design
+Model factory enables developing and publishing ML models at scale by combining MLOps and DevOps pipelines with a standardized convention based approach for creating an efficient and scalable system which allows generating, training, and maintaining ML models.
+
+MLOps pipelines built on top of Azure Machine Learning at its core are responsible for automating and orchestrating various stages of a ML model lifecycle ranging from data pre-processing, feature engineering, model training, hyperparameter tuning, and model evaluation.
+
+DevOps pipelines built on top of Azure DevOps are responsible for packaging the trained ML models into Docker containers and facilitating their deployment across multiple environments.
+
+The workflow of a model from a developer environment to a production container registry is depicted below. This also depicts various validation checks that are enforced in various stages of the workflow. 
+ ![image](/docs/images/MLModelFactory_Design.jpg)
+
+#### Key Features
+•	Supports generation of multiple ML Models
+•	MLOps pipeline for Data preparation, transformation, Model Training, evaluation, scoring and registration.
+•	Each ML Model is packaged in an independent Docker Image
+•	Model verification before storing the Docker image
+•	All Docker images are stored in Azure Container Registry
+•	Builds and deploys Smoke Test module on Edge device
+•	Based on Azure ML SDK v2 1.4 and IoT Edge runtime 1.4
```

Comment:
We can remove this point Based on Azure ML SDK v2 1.4 and IoT Edge runtime 1.4

Reply:
Removed the bullet point